### PR TITLE
update readme.md

### PR DESCRIPTION
**Pull request by dazhou-wish**
Summary
test pull request

Jira
https://test/123

Test Plan
.....

Type of Change

### CODE REVIEW

**Comment by danielzcw** [comment]
您好，您的邮件已收，谢谢！

**Review by dazhou-wish — COMMENTED** [review]
why not use it?

**Comment by dazhou-wish** [comment]
> why not use it?

File: README.md [hunk 1]
```diff
@@ -667,10 +667,6 @@ Executed after deleting a record
 Special thanks to
 -----------------
 
-[mikelbring](https://github.com/mikelbring)
```
Comment (danielzcw) [inline README.md:670]: remove it

File: README.md [hunk 2]
```diff
@@ -667,10 +667,6 @@ Executed after deleting a record
...
-[Paul Hrimiuc](https://github.com/hpaul/)
```
Comment (dazhou-wish) [inline README.md:671]: why?

**Pull request merged**